app/controllers/Application.java

```diff
@@ -1477,12 +1508,40 @@ public static Result restFlowMetricsGraphData(String flowDefId) {
         .order()
         .desc(AppResult.TABLE.FINISH_TIME)
         .setMaxRows(JOB_HISTORY_LIMIT)
-        // The 2nd and 3rd table are not required for plotting the graph
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS, AppHeuristicResult.getSearchFields())
-        //.fetch(AppResult.TABLE.APP_HEURISTIC_RESULTS + "."
-        //    + AppHeuristicResult.TABLE.APP_HEURISTIC_RESULT_DETAILS, "*")
         .findList();
 
     return results;
   }
+
+  private static class AppResourceUsageData {
```

Comment:
You are right. This makes sense. I thought of nested static classes as classes containing static variables but it turns out that they can be instantiated. And in this case, keeping it static might be the right choice to directly access the class without an instance of enclosing class.